src/patternfly/components/Form/form.scss

```diff
@@ -323,6 +316,8 @@ $pf-v5-c-form--m-horizontal--breakpoint-map: build-breakpoint-map("sm", "md", "l
 }
 
 .#{$form}__group-control {
+  display: grid;
+  gap: var(--#{$form}__group-control--Gap);
```

Comment:
I'm not sure if we want to make this a grid layout by default? Currently you can place stuff in it and it will have a normal layout that you can modify. This will make any children grid items and be the same effectively as the `.pf-m-stack` modifier.